include/RAJA/policy/openmp/target_reduce.hpp

```diff
@@ -47,6 +47,39 @@ namespace RAJA
 
 namespace omp
 {
+#ifdef RAJA_ENABLE_TARGET_OPENMP
```

Comment:
I wonder why this is specific for `RAJA::omp` when this is just a common min/max-loc reduction function object. This lived just fine under `RAJA::reducers` before. I'm not sure why it needs moved here

Reply:
Yeah this is the most incongruous piece, where pattern/detail/reduce moved ahead in time wrt to BaseReduceMinLoc and ValueLoc constructs, while target-reduce fell behind. Note, we could generate an new issue to adapt either of these two to work together and gen a separate PR. Those function objects that you notice were pulled from older gen RAJA where it did live in reduce::, and were copied verbatim into RAJA::omp.